chocolatey/tools-online/chocolateyinstall.ps1

```diff
@@ -5,12 +5,17 @@ if (($domainRole -eq 4) -Or ($domainRole -eq 5)) {
   Write-Host "Installation on a Domain Controller is not yet supported - aborting"
   exit -1
 }
+url = "https://s3.amazonaws.com/ddagent-windows-stable/ddagent-cli-$($env:chocolateyPackageVersion).msi"
+if ($env:chocolateyPackageVersion -match "(\d+\.\d+\.\d+)-rc\.(\d+)") {
```

Comment:
It duplicates a bit the logic from https://github.com/DataDog/datadog-agent/blob/master/tasks/winbuildscripts/Generate-Chocolatey-Package.ps1#L26